src/mediatype-icon.ts

```diff
@@ -41,21 +46,44 @@ export class MediatypeIcon extends LitElement {
     film: 'Film',
     image: 'Image',
     movies: 'Movie',
+    radio: 'Radio',
     software: 'Software',
     texts: 'Text',
     tv: 'TV',
     video: 'Video',
     web: 'Web',
   };
 
+  private getDisplayMediaType = () => {
```

Comment:
For things that don't require parameters, I would suggest using the `getter` form:
```
private get displayMediaType() {
  ...
  return ...
}

...

const displayType = this.displayMediaType
```